docs/fr-FR/config/README.md

```diff
@@ -242,7 +242,7 @@ Le `format` par défaut est utilisé pour définir le format de l'invite, si il
 ```toml
```

Comment:
I think these translations come in via some other means, so should not be touched by a PR itself.